indexing-service/src/test/java/org/apache/druid/indexing/overlord/hrtr/HttpRemoteTaskRunnerTest.java

```diff
@@ -1686,7 +1686,7 @@ public void testShutdown()
   @Test(timeout = 60_000L)
   public void testSyncMonitoring_finiteIteration()
   {
-    TestDruidNodeDiscovery druidNodeDiscovery = new TestDruidNodeDiscovery();
+    TestDruidNodeDiscovery druidNodeDiscovery = new TestDruidNodeDiscovery(true);
```

Comment:
lets not change an existing test case like this. It will be confusing to a reader as to why `timedOut` is set to true in this particular test case. lets get rid of it entirely or add a new test case